src/utils/apm/utils.ts

```diff
@@ -52,6 +53,21 @@ export function stipIpfsPrefix(ipfsDirtyHash: string): string {
   )
 }
 
+/**
+ * Returns a joined IPFS location given an IPFS gateway and an IPFS path
+ * This util makes sure the url is properly joined, and that it contains
+ * the "ipfs" route only once, stripping it from the gateway and the location
+ * @param ipfsGateway "https://ipfs.io"
+ * @param location "Qmzz"
+ * @return "https://ipfs.io/ipfs/Qmzz/artifact.json"
+ */
+export function joinIpfsLocation(
+  ipfsGateway: string,
+  location: string
+): string {
+  return urlJoin(stipIpfsPrefix(ipfsGateway), 'ipfs', stipIpfsPrefix(location))
```

Comment:
I think we misspelled `strip` here

Reply:
Offloaded to https://github.com/aragon/buidler-aragon/issues/131